src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/CheckPowTransactionRule.cs

```diff
@@ -48,7 +48,7 @@ public virtual void CheckTransaction(Network network, PowConsensusOptions option
             }
 
             // Size limits (this doesn't take the witness into account, as that hasn't been checked for malleability).
-            if (BlockSizeRule.GetSize(network, tx, NetworkOptions.TemporaryOptions & ~NetworkOptions.Witness) > options.MaxBlockBaseSize)
+            if (BlockSizeRule.GetSize(network, tx, TransactionOptions.All & ~TransactionOptions.Witness) > options.MaxBlockBaseSize)
```

Comment:
I think `TransactionOptions.All & ~TransactionOptions.Witness` is None.

Reply:
yeah correct I reverted back to before the NetworkOptions change good spot.